warehouse/routes.py

```diff
@@ -209,6 +209,9 @@ def includeme(config):
     )
     config.add_route("packaging.file", files_url)
 
+    # SES Webhooks
+    config.add_route("ses.hook", "/_/ses-hook/", domain=warehouse)
```

Comment:
I've not used SES -> SNS -> webhook for this in the past, can we ensure we have the appropriate [DeliveryPolicies](https://docs.aws.amazon.com/sns/latest/dg/DeliveryPolicies.html) in place to give these webhooks a fighting chance?